chia/util/keychain.py

```diff
@@ -219,31 +222,49 @@ def mnemonic_str(self) -> str:
         return " ".join(self.mnemonic)
 
 
+class KeyTypes(str, Enum):
+    G1_ELEMENT = "G1 Element"
```

Comment:
Do we want a type for private BLS key - although I guess we don't actually store the private key - it's the mnemonic. (although maybe we should also store the private key bits, I dunno)

So do we want a type for "mnemonic"?